src/engine/execute.js

```diff
@@ -207,12 +203,7 @@ var execute = function (sequencer, thread) {
             // Find the I/O device and execute the query/function call.
             if (runtime.ioDevices[device] && runtime.ioDevices[device][func]) {
                 var devObject = runtime.ioDevices[device];
-                // @todo Figure out why eslint complains about no-useless-call
-                // no-useless-call can't tell if the call is useless for dynamic
-                // expressions... or something. Not exactly sure why it
-                // complains here.
-                // eslint-disable-next-line no-useless-call
```

Comment:
In this case, the linter was onto something right. `call()` expects `thisArg, arg1, arg2, arg3 ...`, so the line could have been replaced by `devObject[func](args);`. But I think what I was going for was not `call` but `apply`, which is `thisArg, [arg]`. No complaints about apply.